.github/workflows/build.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       # Python versions in sync with the ones in test.yml.
       matrix:
         os:
-          - "macos-11"
+          - "macos-12"
```

Comment:
GitHub says that `macos-14` will use M1 macs, if that is useful.

https://github.blog/changelog/2024-01-30-github-actions-introducing-the-new-m1-macos-runner-available-to-open-source/

Reply:
Ah, that's good to know... assuming we can build a wheel that works on older macOS. Which cibuildwheel might automate. Or not.